brianlong/www.validators.app
# 184004267 commission tracking script

#### What's this PR do?
- check get_inflation_reward endpoint for the commission changes we've not caught before
- run the check an hour after new epoch was created
- add missing commission changes

#### How should this be manually tested?
- run tests
- populate db with current and previous EpochWallClock
- try running the service in console
- check if new CommissionHistory records are created (it should have from_inflation_rewards: true)

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184004267)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
